src/ResourceManager/DataMigration/Commands.DataMigration/help/AzureRM.DataMigration.md

```diff
@@ -28,12 +28,18 @@ Locale: en-US
 ### [New-AzureRmDataMigrationProject](New-AzureRmDataMigrationProject.md)
 {{Creates a new Azure Database Migration Service project.}}
 
+### [New-AzureRmDataMigrationFileShare](New-AzureRmDataMigrationFileShare.md)
+{{Creates the File share object for the Azure Database Migration Service, which specifies the file share information for migration.}}
+
 ### [New-AzureRmDataMigrationService](New-AzureRmDataMigrationService.md)
 {{Creates a new instance of the Azure Database Migration Service.}}
 
 ### [New-AzureRmDataMigrationSqlServerSqlDbSelectedDB](New-AzureRmDataMigrationSqlServerSqlDbSelectedDB.md)
 {{Creates a MigrateSqlServerSqlDbDatabaseInput object that contains information about source and target databases for migration.}}
 
+### [New-AzureRmDataMigrationSqlServerSqlDbMiSelectedDB](New-AzureRmDataMigrationSqlServerSqlDbMiSelectedDB.md)
```

Comment:
I think this needs to be regenerated - this is no longer the name.

Reply:
Fixed.